app/data_access/control_data.py

```diff
@@ -190,6 +191,10 @@ class Meta:
     siren = graphene.String()
     company_address = graphene.String()
     mission_address_begin = graphene.String()
+    current_business_type = graphene.Field(
```

Comment:
Shouldn't it be named `business_type_during_control` or `business_type_at_control` because `current` may be confusing?